lib/percona_migrator/lhm_parser.rb

```diff
@@ -1,53 +0,0 @@
-module PerconaMigrator
```

Comment:
I'll use it as an optional adapter that can specify in config. In case you have LHM migrations (like us) plug it in to parse LHM statements first to then handle it to the percona connection adapter. If not, proceed with the regular percona connection adapter.